### ISSUE TITLE
File should not be exectuable

### ISSUE THREAD

**Issue by justinmeiners** [ruimarinho/bitcoin-core#91]
The following file is marked executable and I don't believe it needs to be:
```
/dist/src/errors/rpc-error.js
```

To fix:
```
chmod -x  rpc-error.js
```

**Comment by sskender** [comment]
Those are not all the files, there are many more that are executable for some reason. I am not sure why.

**Comment by justinmeiners** [comment]
indeed. It's little, but basic, mistakes that make me concerned about the security of this project.

**Comment by ruimarinho** [comment]
@justinmeiners pull requests are open so you contribute to the security of this module by submitting patches - I'd appreciate that. Are you able to demonstrate an attack based on having these files as executables?

**Comment by justinmeiners** [comment]
> pull requests are open

I am no longer doing work related to this. This issue was filed more than a year ago.

**Comment by sskender** [comment]
> @justinmeiners pull requests are open so you contribute to the security of this module by submitting patches - I'd appreciate that. Are you able to demonstrate an attack based on having these files as executables?

I have just created a PR so feel free to audit it.
